.vib/minio/cypress/cypress/integration/minio_spec.js

```diff
@@ -1,67 +1,69 @@
 /// <reference types="cypress" />
-import {
-  random,
-} from './utils'
+import { random } from './utils';
 
 it('allows user to log in and log out', () => {
   cy.login();
   cy.get('[data-testid="ErrorOutlineIcon"]').should('not.exist');
-  cy.get("#logout").scrollIntoView().should('be.visible').click();
-})
+  cy.get('#logout').scrollIntoView().click();
+});
 
-it('allows creating a bucket and file upload', () => {  
+it('allows creating a bucket and file upload', () => {
   cy.login();
   cy.visit('add-bucket');
-  cy.contains('div','Return to Buckets')
   cy.fixture('testdata').then((td) => {
-    cy.get('#bucket-name').should('be.visible').type(`${td.bucketTitle}.${random}`);
-  })
-  cy.contains('button[type="submit"]','Create Bucket').should('be.visible').click();
+    cy.get('#bucket-name')
+      .should('be.visible')
+      .type(`${td.bucketTitle}.${random}`);
+  });
+  cy.contains('button[type="submit"]', 'Create Bucket')
+    .should('be.visible')
+    .click();
   cy.fixture('testdata').then((td) => {
-    cy.url().should('include',`buckets/${td.bucketTitle}`);
-  })
+    cy.url().should('include', `buckets/${td.bucketTitle}`);
+  });
 
-  cy.get('#upload-main').should('be.visible').click();
-  cy.contains('span','Upload File').should('be.visible').click();
-  cy.get('div#object-list-wrapper > input[type="file"]').should('not.be.disabled').selectFile('cypress/fixtures/example.json',
-  {force:true});
-  cy.get('button[aria-label="Clear Completed List"]').should('be.visible').click();
-  cy.contains('div','Object uploaded successfully.').should('be.visible');
-})
+  cy.contains('#upload-main', 'Upload').click();
+  cy.contains('span', 'Upload File').should('be.visible').click();
+  cy.get('div#object-list-wrapper > input[type="file"]')
+    .should('not.be.disabled')
+    .selectFile('cypress/fixtures/example.json', { force: true });
+  cy.reload();
+  cy.contains('example.json');
+});
```

Comment:
Not a big concern, but this might no longer ensure that the test would be valid for multiple executions. If there is an error while uploading the file and then we reload the page, would we still see the `example.json` on the page?

Before we looked for the `Object uploaded successfully` notification.

Reply:
I understand this, however, in the case of MiniO we upload the file to a bucket we create on the fly. Hence, if we run the same test on the same server twice, the files will anyway be uploaded to different buckets so they will never coincide. 
![MiniO](https://user-images.githubusercontent.com/47592680/163125083-027ee5bd-5dbd-4707-bd6f-26f8bc1d3721.png)

Reply:
Fair enough! 😁